src/test/java/io/aiven/kafka/connect/gcs/GcsSinkTaskTest.java

```diff
@@ -396,6 +478,44 @@ private SinkRecord createRecord(final String topic,
             TimestampType.CREATE_TIME);
     }
 
+    private Iterable<Header> createHeaders() {
+        final Random random = new Random();
```

Comment:
Generally it's better to use deterministic inputs unless there is a way to reproduce a failed case, but let's not change only for this here.